apps/admin/frontend/src/app.test.tsx

```diff
@@ -408,9 +418,21 @@ test('tabulating CVRs', async () => {
 
   await apiMock.authenticateAsElectionManager(eitherNeitherElectionDefinition);
 
+  expectReportsScreenCardCountQueries({
+    apiMock,
+    isPrimary: false,
+  });
+  apiMock.expectGetManualResultsMetadata([]);
+  apiMock.expectGetScannerBatches([
+    // need to mock scanner batches to get link button to batch report
+    {
+      electionId: 'election-id',
+      batchId: '2',
+      label: 'Batch 2',
+      scannerId: 'scanner-id',
+    },
+  ]);
```

Comment:
Some of these tests at the app level are pretty clunky. When we transition tally reports (next piece of work) they will have no purpose and will be removed.